config/prow/cluster/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -86,7 +87,19 @@
           {
             alert: 'ghproxy-global-status-code-4xx',
             expr: |||
-              sum(rate(github_request_duration_count{status=~"4..",status!="404",status!="410"}[30m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[30m])) * 100 > 3
+              sum(rate(github_request_duration_count{status=~"4..",status!="404",status!="410",status!="403",status!="405",status!="422"}[30m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[30m])) * 100 > 3
+            |||,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: '{{ $value | humanize }}%% of all API requests through the GitHub proxy are errorring with code {{ $labels.status }}. Check %s.' % [monitoringLink('/d/%s/github-cache?orgId=1&refresh=1m&fullscreen&panelId=8' % [dashboardID], 'the ghproxy dashboard')],
+            },
+          },
+          {
+            alert: 'ghproxy-global-status-code-403-405-422',
+            expr: |||
+              sum(rate(github_request_duration_count{status=~"403|405|422"}[30m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[30m])) * 100 > 10
```

Comment:
Hopefully we can tighten the time window for the global alerts more than this now that we are excluding the noisy path+status combos. 30mins is a very long delay for an alert about this kind of problem.